js/js-loops/js-loops_forof-list/js/index.js

```diff
@@ -18,5 +18,12 @@ const programmingLanguages = [
 ];
 
 // --v-- write/change code here --v--
+for (const list of programmingLanguages) {
+  const li = document.createElement("li");
+  li.textContent = list;
 
+  main.appendChild(li);
+}
+
+main.appendChild(ol);
```

Comment:
i think you just need to append the list item to the ol, not the main ie ol.append(li)